src/pgrn-query-expand.c

```diff
@@ -471,6 +472,52 @@ PGrnFindTermAttributeNumber(PGrnQueryExpandData *data,
 	return;
 }
 
+static const char*
+PGrnConvertRelationKind(char relkind)
+{
+	const char *relationKind;
+	switch(relkind) {
```

Comment:
It seems that we should prepare `clang-format`...

```suggestion
	switch (relkind)
	{
```